src/GafferOSL/OSLShader.cpp

```diff
@@ -659,30 +659,38 @@ template <typename PlugType>
 Plug *loadSplineParameters( const OSLQuery::Parameter *positionsParameter, const OSLQuery::Parameter *valuesParameter, const OSLQuery::Parameter *basisParameter, const InternedString &name, Gaffer::Plug *parent )
 {
 	typedef typename PlugType::ValueType ValueType;
-	ValueType defaultValue;
 
 	const std::string &basis = basisParameter->sdefault.front().string();
+	typename PlugType::Interpolation defaultInterpolation = PlugType::CatmullRom;
+	// TODO - should we keep low level support of Bezier even if we don't want to expose it in the UI?
```

Comment:
Maybe remove it? It can't have been used successfully in the past, so I don't see any harm in removing it...